doc/Administration-guide/System-installation/index.md

```diff
@@ -5,39 +5,52 @@ permalink: /:path/
 nav: /:path/Administration-guide/System-installation/
 sort_idx: 20
 ---
-* [Introduction](#introduction)
-* [System requirements](#system-requirements)
-* [Supported OS](#supported-os)
-* [Third party components](#third-party-components)
-  * [Zookeeper](#zookeeper)
-  * [SQL database](#sql-database)
-  * [NoSQL database](#nosql-database)
-* [Installing Kaa](#installing-kaa)
-  * [Single node installation](#single-node-installation)
-  * [Node cluster setup](#node-cluster-setup)
-  * [AWS deployment](#aws-deployment)
-  * [Planning your deployment](#planning-your-deployment)
-* [Troubleshooting](#troubleshooting)
+
+* TOC
+{:toc}
 
 ## Introduction
 
 This guide overview over kaa platform installation.
 
 ## System requirements
 
-To use Kaa, your system must meet the following minimum system requirements.
+To use Kaa, your system must meet the following minimum system requirements:
 
    * 64-bit OS
+   
+   for single kaa-node
```

Comment:
for single kaa-node service with all third-party services (SQL, NoSQL, Zookeeper, etc.) deployed remotely